modules/preview-react/select/stories/stories.tsx

```diff
@@ -216,4 +233,15 @@ storiesOf('Preview/Select/React/Left Label', module)
     >
       {controlComponent(<Select name="contact" options={options} />)}
     </FormField>
+  ))
+  .add('Default with Custom Render Selected Method', () => (
+    <FormField
+      labelPosition={FormField.LabelPosition.Left}
+      label="Label"
+      inputId="select-default-custom-selected"
+    >
+      {controlComponent(
+        <Select name="icon" options={customOptions} renderSelected={customRenderSelectedOption} />
+      )}
+    </FormField>
```

Comment:
Same feedback as above with the Top Label story (consolidate story into Left Label > "Default with Custom Options Data").